### PR TITLE
Add 'How to Use' section to README.md; remove unnecessary modules in …

### PR DESCRIPTION
**Pull request by wmoustafa**
…examples code

### CODE REVIEW

File: transportable-udfs-documentation/transport-udfs-presto.patch [hunk 1]
```diff
@@ -0,0 +1,139 @@
+diff --git a/presto-main/pom.xml b/presto-main/pom.xml
```
Comment (cwsteinbach) [inline transportable-udfs-documentation/transport-udfs-presto.patch]: Was this file included by accident?

File: transportable-udfs-examples/settings.gradle [hunk 2]
```diff
@@ -7,11 +7,7 @@ rootProject.name = 'transportable-udfs-examples'
 includeBuild('../')
 
 def modules = [
-    'transportable-udfs-example-udfs-avro',
-    'transportable-udfs-example-udfs',
-    'transportable-udfs-example-udfs-hive',
-    'transportable-udfs-example-udfs-presto',
-    'transportable-udfs-example-udfs-spark'
+    'transportable-udfs-example-udfs'
```
Comment (cwsteinbach) [inline transportable-udfs-examples/settings.gradle]: Consider changing the module name to transportable-udfs-examples.

File: transportable-udfs-documentation/transport-udfs-presto.patch (hunk 1 above)
Comment (wmoustafa) [reply transportable-udfs-documentation/transport-udfs-presto.patch]: No. This is a patch that needs to be applied to the Presto engine to enable it to accept SqlScalarFunction classes as plugins. We create SqlScalarFunction presto UDFs.

File: transportable-udfs-examples/settings.gradle (hunk 2 above)
Comment (wmoustafa) [reply transportable-udfs-examples/settings.gradle]: Will do in a separate patch.

**Pull request merged**